#135 Fix: ColumnasOcultasNoRecuperables

DiegoOrellanaVaca opened the pull request:
Solucioné el bug almacenando la visibilidad de las columnas en un estado y forzando que por lo menos una columna siempre esté activa. De esa forma, aunque se intente deseleccionar todas, al menos esa columna permanece visible.

DiegoOrellanaVaca committed:
Solucioné el bug almacenando la visibilidad de las
columnas en un estado y forzando que por lo menos
una columna siempre esté activa. De esa forma,
aunque se intente deseleccionar todas, al menos
esa columna permanece visible.

ObiTobi64 commented on src/pages/Student/StudentsPage.tsx:
```diff
@@ -170,15 +178,15 @@ const StudentPage = () => {
+            disabled={!addStudentPermission}
+          >
+            Agregar Estudiante
+          </Button>
```
Evitar espacios adicionales que no aporten a la solución del código

ObiTobi64 commented on src/pages/Student/StudentsPage.tsx:
```diff
@@ -192,9 +200,22 @@ const StudentPage = () => {
             }}
+            columnVisibilityModel={columnVisibilityModel}
+            onColumnVisibilityModelChange={(newModel) => {
+              // Forzamos que la columna "name" (Nombre Completo) siempre esté visible
```
Elimina este comentario

ObiTobi64 commented on src/pages/Student/StudentsPage.tsx:
```diff
@@ -30,14 +30,23 @@ const StudentPage = () => {
   const [viewStudentReportPermission, setViewStudentReportPermission] = useState<Permission>();
-  
+
+  // Estado para el modelo de visibilidad de columnas
```
Elimina este comentario

DiegoOrellanaVaca committed:
Solucioné el bug almacenando la visibilidad de las
columnas en un estado y forzando que por lo menos
una columna siempre esté activa. De esa
forma, aunque se intente deseleccionar todas, al
menos esa columna permanece visible.

ObiTobi64 reviewed (commented):
LGTM
